src/types/markers.js

```diff
@@ -659,6 +659,10 @@ export type IPCMarkerPayload_Gecko = {|
   // Phase is not present in older profiles (in this case the phase is "endpoint").
   phase?: 'endpoint' | 'transferStart' | 'transferEnd',
   sync: boolean,
+  // `tid` of the thread that this marker is originated from. It is undefined
+  // when the IPC marker is originated from the same thread. Also, this field is
+  // added in Firefox 100. It will always be undefined for the older profiles.
+  tid?: Tid,
```

Comment:
I'm confused, because the property name in the gecko patch doesn't look to be that one?

![image](https://user-images.githubusercontent.com/454175/159251350-fd1f84cc-84f4-4458-a8c4-c921435e9d03.png)

Reply:
Oh sorry, this is a mistake on the front-end side. Updated the patch now.